subprojects/libpng.wrap

```diff
@@ -0,0 +1,12 @@
+[wrap-file]
+directory = libpng-1.6.37
+source_url = https://github.com/glennrp/libpng/archive/v1.6.37.tar.gz
+source_filename = libpng-1.6.37.tar.gz
+source_hash = ca74a0dace179a8422187671aee97dd3892b53e168627145271cad5b5ac81307
+patch_url = https://wrapdb.mesonbuild.com/v1/projects/libpng/1.6.37/2/get_zip
+patch_filename = libpng-1.6.37-2-wrap.zip
+patch_hash = c9dde143db819e803c9d59628863a9c8ea34f84b7e8e28e3d881502a5747ec56
+
+[provide]
+libpng = libpng_dep
+
```

Comment:
General comment about everything in `subprojects/`:

I'm not fond of adding pointers to fixed version of our dependencies to the repo. We would need to closely monitor their releases or at least CVEs to avoid the default build creating binaries vulnerable to known security flaws. This is even more problematic with wrapDB deps, as here fixed versions might not even be available for a long time. *(Until recently brotli and expat in wrapDB were years outdated and affected by several CVEs. It appears there's no monitoring of upstream releases or CVEs and the wrapDB overview page also doesn't indicate what the latest upstream release is)*. 
 
If they were included in tarballs, it’s even more problematic and we'd need to create new releases/tarballs not just for new libass releases but also to fix CVes in dependencies. Otherwise the pointers would be useless there as they cannot be safely used.

Reply:
I have partially implemented nvchecker release scraping for the WrapDB in a local commit from September... I really should finish wiring everything up I guess.

It's not strictly necessary to check them into git, anyway. It's convenient, that's all. The `meson wrap install` command can install wraps that the user wants to use, at which point it works without any changes to meson.build; `meson wrap update` can update them (only to the latest WrapDB version, which is what you'd want anyway). Both are manual steps so if your goal is to have it all work OOTB then checking it into git might be interesting. Some people want this; some people don't.

Reply:
If it isn't required, then not adding any `*.wrap` files to the repo would be my preferred solution.